### PR TITLE
Forsaken being spawned now swaps all groundside resin to forsaken hive.

### PR DESCRIPTION
**Pull request by NewyearnewmeUwu**
# About the pull request
Forsaken are being slowed down by normal xeno hive weeds and killed by the normal hive's groundside acid traps. This fixes that. It also deletes normal hive groundside acid pillars as a TEMPORARY WORKAROUND because they will delete forsaken weeds even when swapped to forsaken hive.

# Explain why it's good for the game
This was requested by Morrow, and it works.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Forsaken being spawned now swaps all groundside resin to forsaken hive.
/:cl:

### CODE REVIEW

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/__DEFINES/dcs/signals/signals_global.dm [hunk 1]
```diff
@@ -50,3 +50,6 @@
 
 /// From /datum/admins/proc/force_predator_round()
 #define COMSIG_GLOB_PREDATOR_ROUND_TOGGLED "!predator_round_toglged"
+
+// Used to convert all groundside weeds, etc to Forsaken hive.
```
Comment (fira) [inline code/__DEFINES/dcs/signals/signals_global.dm:54]:
```suggestion
/// From /datum/game_mode/colonialmarines/proc/check_ground_humans()
```

**Review by fira — APPROVED** [review]

**Comment by ihatethisengine** [comment]
I am not sure if it should change all the weeds. Yesterday we had a really fun round, when I was playing forsaken and two xenos from regular sky came back on a pod. It was a tense fight, partly because they still had their walls, sticky and weeds.

File: code/game/gamemodes/colonialmarines/colonialmarines.dm [hunk 2]
```diff
@@ -273,6 +273,7 @@
 
 	if(groundside_humans > (groundside_xenos * GROUNDSIDE_XENO_MULTIPLIER))
 		SSticker.mode.get_specific_call("Xenomorphs Groundside (Forsaken)", FALSE, FALSE)
+		SEND_GLOBAL_SIGNAL(COMSIG_GLOB_GROUNDSIDE_FORSAKEN_HANDLING)
```
Comment (morrowwolf) [inline code/game/gamemodes/colonialmarines/colonialmarines.dm:276]: This really shouldn't need to be sent more than once on hijack itself. There's likely a better place to send (best guess, wherever it resets the hive core).

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/colonialmarines/colonialmarines.dm (hunk 2 above)
Comment (ihatethisengine) [reply code/game/gamemodes/colonialmarines/colonialmarines.dm]: Just do that on hijack as I suggested

**Comment by Segrain** [comment]
This should happen _after_ the dropship takeoff and it should skip the weeds on the dropship itself.
Frankly, much simpler solution would be to simply hardcode regular and forsaken hive as allies. Even should a regular xeno somehow end back on the planet, why should they be hostile to each other?

**Pull request closed without merging**